docs/DeveloperGuide.adoc

```diff
@@ -14,7 +14,7 @@ ifdef::env-github[]
 endif::[]
 :repoURL: https://github.com/CS2103-AY1819S1-F11-2/main
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team Concierge(TM)`      Since: `Jun 2016`      Licence: `MIT`
```

Comment:
I'm actually not sure if we have to credit the other team. Anyway, should be no issue with this.

The date can be changed to Aug 2018 too.

Reply:
We credited in the README already, so i think should be fine? 😃 

updated in latest commit